MREUnityRuntime/MREUnityRuntimeLib/Assets/AssetLoader.cs

```diff
@@ -146,125 +146,153 @@ private async Task LoadAssets(LoadAssets payload, Action onCompleteCallback)
 			onCompleteCallback?.Invoke();
 		}
 
+		// Global: Keep track of the number of active glTF web requests.
+		private static int activeLoads = 0;
+
 		private async Task<IList<Asset>> LoadAssetsFromGLTF(AssetSource source, Guid containerId, ColliderType colliderType)
 		{
+			// Spin asynchronously until activeLoads allows us through.
+			while (activeLoads >= 4)
```

Comment:
wbn to have the '4' as a shared constant for easy tuning / readability.

Reply:
Good call. Updated.